database/factories/ChannelFactory.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\Channel;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class ChannelFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = Channel::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+            'name' => $this->faker->sentence(),
+            'description' => $this->faker->paragraph(),
+            'slug' => $this->faker->sentence(),
```

Comment:
slugs are of a different case. Slugs follow snake case so it should be like so: Str::slug($this->faker->sentence());